### PR TITLE
CASSANDRA-19896 5.0 set executable flag for .build/docker shell scripts i…

### PR DESCRIPTION
**Pull request by smiklosovic**
…n source artifact

Thanks for sending a pull request! Here are some tips if you're new here:
 
 * Ensure you have added or run the [appropriate tests](https://cassandra.apache.org/_/development/testing.html) for your PR.
 * Be sure to keep the PR description updated to reflect all changes.
 * Write your PR title to summarize what this PR proposes.
 * If possible, provide a concise example to reproduce the issue for a faster review.
 * Read our [contributor guidelines](https://cassandra.apache.org/_/development/index.html)
 * If you're making a documentation change, see our [guide to documentation contribution](https://cassandra.apache.org/_/development/documentation.html)
 
Commit messages should follow the following format:

```
<One sentence description, usually Jira title or CHANGES.txt summary>

<Optional lengthier description (context on patch)>

patch by <Authors>; reviewed by <Reviewers> for CASSANDRA-#####

Co-authored-by: Name1 <email1>
Co-authored-by: Name2 <email2>

```

The [Cassandra Jira](https://issues.apache.org/jira/projects/CASSANDRA/issues/)

### CODE REVIEW

File: build.xml [hunk 1]
```diff
@@ -978,6 +978,7 @@
           <exclude name="bin/*.in.sh" />
           <include name="tools/bin/*"/>
           <exclude name="tools/bin/*.in.sh" />
+          <include name=".build/docker/*.sh"/>
```
Comment (michaelsembwever) [inline build.xml]: what about the `.build/*.sh` scripts ?

Comment (smiklosovic) [reply build.xml]: not sure if it is smart like that, will check

Comment (smiklosovic) [reply build.xml]: @michaelsembwever what you suggested does not work.

Comment (smiklosovic) [reply build.xml]: `.build/**/*.sh` seems to work. I've changed it to that.

**Review by michaelsembwever — APPROVED** [review]

**Pull request closed without merging**